crates/sui-framework/packages/move-stdlib/sources/vector.move

```diff
@@ -158,6 +158,40 @@ module std::vector {
         v.pop_back()
     }
 
+    /// Take all elements of the vector `v` except the first `n` elements. Aborts if `n` is greater
+    /// than the length of `v`. Modifies the original vector.
+    public fun skip_mut<T>(v: &mut vector<T>, n: u64): vector<T> {
```

Comment:
At first, I thought maybe this was backwards and this should be `take_mut`, but I think this makes sense: you want to return the desired elements. In short, you want the return values of `skip|take` to match with `(skip|take)_mut`